### ISSUE TITLE
Article IDs off by one

### ISSUE THREAD

**Issue by mahagr** [fastslack/jUpgrade#22]
I migrated A site with ~300 articles from J1.5.25 to J2.5.1 and it looks like almost every article has id+1 in the new site.

1,2,3,4,6,7,9,10... => 1,2,3,4,6,7,10,11...

So ID 9 and larger has wrong ID.

**Comment by fastslack** [comment]
This is very strange... are you running jUpgrade with this patch? 7b18b0f16f727bc0ea19797ab96be95becf52729

**Comment by mahagr** [comment]
Oops, looks like i accidentally installed the old version. Let me try again.

**Comment by mahagr** [comment]
The issue has been fixed.
